### PR TITLE
Adding automatically escaped in print scriptlet.

### LINKED ISSUE #7: <%= x %> x is automatically escaped

### PR DESCRIPTION
**Pull request by caiooliveiraeti**
I want to close #7

### CODE REVIEW

**Comment by Turini** [comment]
that sounds good to me, thanks @caiooliveiraeti! 
I'll wait a few more time before merging, so @guilhermesilveira may take a look on it too

**Comment by guilhermesilveira** [comment]
Great! Thanks @caiooliveiraeti I've added one comment on the name of a boolean variable. I've just one question about the line where the escaping occur. The way it was written, it will escape during compile time, while we need to escape during runtime, as in:

```
<%= user.getDescription() %>
```

This should escape the user description, by translating the above to:

```
write(escape(user.getDescription()));
```

Sorry that the issue did not give that detail... if you need feel free to ask for more details in any (and even more than one) issue.

I think that should be all!!

Thanks!

**Comment by caiooliveiraeti** [comment]
Hi @guilhermesilveira,

I just made ​​the commit to change the variable name, was better. Thanks for the tip.

**Comment by Turini** [comment]
@guilhermesilveira it's already escaping during runtime, check [this test](https://github.com/caelum/vraptor-panettone/pull/72/files#diff-a955326f2b419aab0545a797244ef049R82).
Can you think in any other case that was not covered in this PR tests? If 
not, I think we can merge here. @caiooliveiraeti thank you again for this PR

**Comment by guilhermesilveira** [comment]
@Turini an integration test where the template generates the code with the raw and without the raw text would be great. That's all. Something on the TemplateTest?
@caiooliveiraeti I saw it now, sorry. Great! THanks!

**Comment by guilhermesilveira** [comment]
ps: you can submit the integration one after merging, no problem!

File: src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java [hunk 1]
```diff
@@ -19,7 +14,8 @@
 	public Variable[] getExtraInjections() {
 		return new Variable[] {
 				new Variable("javax.servlet.http.HttpServletResponse", "res"),
-				new Variable("br.com.caelum.vraptor.Result", "result")};
+				new Variable("br.com.caelum.vraptor.Result", "result"),
+				new Variable("br.com.caelum.vraptor.panettone.api.PrintEscaper", "printEscaper")};
```
Comment (caiooliveiraeti) [inline src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java]: @guilhermesilveira I have a doubt. The declaration of this variable is in the correct place ? Or should be in PanettoneWalker?

Comment (guilhermesilveira) [inline src/main/java/br/com/caelum/vraptor/panettone/VRaptorCompilationListener.java]: This is the right place in the current implementation, we might change it a
little bit if rolling back the interface implenetation next week, but don't
worry until then

**Comment by Turini** [comment]
> an integration test where the template generates the code with the raw and 
without the raw text would be great. That's all. Something on the TemplateTest?

That sounds great. I'll merge here as you suggested, and take o look on this test. 
@caiooliveiraeti do you want to work on this? Just let me know, if not, I can do it.

**Comment by Turini** [comment]
Thanks again, @caiooliveiraeti! 
Your contributions have been extremely valuable for this project

**Comment by guilhermesilveira** [comment]
@caiooliveiraeti  great! ready for the next one? :)

**Comment by caiooliveiraeti** [comment]
@guilhermesilveira I'm ready. ;-)

**Pull request merged**